doc/api/tls.md

```diff
@@ -911,6 +909,21 @@ socket.on('end', () => {
 });
 ```
 
+Note from the above example that if you don't wrap the `net.Socket` in a
```

Comment:
I don't understand what the paragraph is trying to say, or the example below is trying to show. What are you trying to describe?

You mention the "above example", but it doesn't create net sockets and wrap them, so how is this a note about the  above example?

Are you trying to describe the somewhat weird behaviour that you can call tls.connect() before calling sock.connect(), with wrapped sockets, and it all works out?

Or are you just trying to give an example of using the `socket:` option to `tls.connect()`, since that option isn't used in the example?

Or something else?